core/src/banking_stage/latest_unprocessed_votes.rs

```diff
@@ -87,10 +87,16 @@ impl LatestValidatorVotePacket {
             Ok(vote_state_update_instruction)
                 if instruction_filter(&vote_state_update_instruction) =>
             {
-                let &pubkey = message
```

Comment:
starry pointed out that this method of identifying the voter is brittle, as it relies on the node id being the first signer.
Grabbing the vote pubkey instead should be safe, as it is required to be the first account on the instruction:
https://github.com/anza-xyz/agave/blob/18059246e5e845855a2d4f8f0441a4fabd101ca6/sdk/program/src/vote/instruction.rs#L153-L154